src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -2318,40 +2328,43 @@ This has significant advantages.  It preserves the _form_ of the code, making th
 than replacement with a function such as `is_gt_zero`.  It also reduces the number of new comparison APIs
 a user must learn: `Zero` handles them all.
 
-`Zero` has a few downsides though:
-
-1. CTAD can't deduce the quantity type as there is no unit provided. So if we try to refactor the
-   following:
-
-    ```cpp
-    quantity<si::metre / si::second> q1 = 0 * m / s;  // OK
-    quantity q2 = 0 * m / s;                          // OK
-    ```
-
-   only the first one will work and the second one will fail to compile:
+`Zero` has one downside: it will not work when passed across _generic quantity_ interfaces.  `Zero`'s value
+comes in situations where the surrounding context makes it unambiguous which quantity type it should
+construct.  While it converts to any _specific_ quantity type, it is not itself a quantity.  This could
+confuse users.
 
-    ```cpp
-    quantity<si::metre / si::second> q1 = ZERO;  // OK
-    quantity q2 = ZERO;                          // Compile-time error
-    ```
```

Comment:
I wouldn't like to remove this. Notice that in nearly all our examples in P2980 we use CTAD and not `auto` for the type. This is a great practice that I use everywhere now. It is good to state explicitly that it does not work with ZERO (even if the reasons for it are obvious).

Reply:
I just cannot imagine a user ever writing something like this.  Like, what is even going through their head?  What did they expect to get?

It's true that we use CTAD instead of `auto`, and it's a great practice.  But it doesn't logically follow that somebody would ever be motivated to use CTAD for _this_.

I think adding examples with no motivation only adds noise to a long paper.